agent/installer/installer_test.go

```diff
@@ -128,6 +128,52 @@ var _ = Describe("Byohost Installer Tests", func() {
 			Expect(func() { NewPreviewInstaller("Ubuntu_20.04.3_x86-64", nil) }).NotTo(Panic())
 		})
 	})
+	Context("When supported OS version is passed for precheck", func() {
+		It("Should not return error", func() {
+			err := supportedOSversion("Ubuntu_20.04.1_x86-64")
```

Comment:
Currently this test is marked with `nolint` which needs to be removed in future. Can we avoid testing private functions? Can it not be via the public method that in turn calls this function?
Because the agent code has no direct access to this function.